.travis.yml

```diff
@@ -46,7 +46,7 @@ matrix:
         - cargo test --examples
         # remove cached documentation, otherwise files from previous PRs can get included
         - rm -rf target/doc
-        - cargo doc --no-deps --all --all-features
+        - cargo doc --no-deps --all --features=std
```

Comment:
To be safe I think it will be better to use `--features="std log"`.

Reply:
Fixed